index.html

```diff
@@ -20,7 +19,7 @@
 
     <body>
         <div id="app"></div>
-        <script>Raven.config('https://23e9c0c37a3841b8a692c98b8978f515@sentry.io/147247').install();</script>
         <script src="scripts/src/app.js"></script>
+        <script>Raven.config('https://23e9c0c37a3841b8a692c98b8978f515@sentry.io/147247').install();</script>
```

Comment:
Since we're using webpack to bundle modules and libraries, it should really be loaded it through the source map for performance gains.